cmd/internal/cli/key_remove.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/sylabs/singularity/docs"
+	"github.com/sylabs/singularity/internal/pkg/sylog"
+	"github.com/sylabs/singularity/pkg/sypgp"
+)
+
+func init() {
+	KeyRemoveCmd.Flags().SetInterspersed(false)
+}
+
+// KeyRemoveCmd is `singularity key remove <fingerprint>' command
+var KeyRemoveCmd = &cobra.Command{
+	Args:                  cobra.ExactArgs(1),
+	DisableFlagsInUseLine: true,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		err := sypgp.RemovePubKey(fmt.Sprintf("%s", args[0]))
+		if err != nil {
+			sylog.Fatalf("Unable to remove public key: %v", err)
```

Comment:
in this case it's better to use "%s" instead of "%v" because "%v" might, in some cases, dump the structure of the variable (not all errors are strings underneath).